dashboard/app/dsl/evaluation_multi_dsl.rb

```diff
@@ -0,0 +1,10 @@
+class EvaluationMultiDSL < MultiDSL
+  def answer(text, weight: weight = 1, stage_name: stage_name = nil)
```

Comment:
wonder if this should be stage_title to differentiate it from what would be a formal, uniquely-identifiable name.